### PR TITLE
Contact Form Email validation

### PR DESCRIPTION
**Pull request by telanet**
# simple email  validation:
The content of the email field has to be in lower case.
When the user submits the form check if the email is in lower case.
If the validation is OK, the form is sent.
If the validation is not OK, show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @telanet, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ The message rendered correctly 

## Required Changes ♻️

- [Required]
As you can see in the screenshot below the `error message` is not displayed bellow the email input, Kindly try to change the place of this error message, so the user understand that the error in the email and not in the text area field.
real

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional]

Try to write meaningful and descriptive commits messages 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const email = document.getElementById('email');
+  if (email.value.trim() === '') {
+    error.innerHTML = 'This field cannot be empty';
+    return false;
+  }
+  if (email.value !== email.value.toLowerCase()) {
+    error.innerHTML = `Your email is not in lowercase. use ${email.value.toLowerCase()} instead`;
+    return false;
+  }
+  return true;
```
Comment (t-yanick) [inline validator.js:13]: - [ ] Please as it is a requirement for this project, kindly ensure the error message is showing up close to the `submit` button. See screenshots below for clarifications. 

errorMessage

errorMessage2

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @telanet, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @Blantraders , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**